doc-Introduction_to_Self_Service_Portal/topics/My_Services_Tab.adoc

```diff
@@ -1,24 +1,95 @@
+
+Follow this procedure to schedule retirement for a service at a later date. For retiring a service immediately, see xref:retiring-service[].
+
+. From the *Services* screen, click on the required service you want to schedule retirement for.
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Set Retirement Dates*. The *Schedule Service Retirement* window will appear.
+image:ssui-schedule-service-retirement.png[]
+. Set the *Retirement Date*.
+. Select the *Retirement Warning* from the list.
+. Click *Save*.
+
+[[retiring-service]]
+=== Retiring a Service
+
+Follow this procedure to retire a service immediately. For retiring a service at a later date, see xref:schedule-retirement[].
+
+. From the *Services* screen, click on the service you want to retire now.
+. Click the image:ssui-retirement-button.png[Service Retire Button] button located on the upper-right corner of the screen, then select *Retire*. The *Retire Service Now* window will appear to confirm your action.
+image:ssui-retire-service-now.png[]
+. Click *Yes, Retire Service Now*.
+
+=== Setting Ownership of a Service
+
+Follow this procedure to set ownership of a service. 
+
+. From the *Services* screen, click on more options next to the service you want set ownership for, then click *Set Ownership*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Set Ownership*.
+image:ssui-set-service-ownership.png[]
+. Select an owner from the list.
+. Select a group from the list.
+. Click *Save*.
+
+=== Adding / Editing Tags
+
+Follow this procedure to add or edit tags for a service. 
+
+. From the *Services* screen, click on more options next to the service you want add or edit tags for, then click *Edit Tags*.
+. Alternatively, click on the service. Click the image:ssui-edit-tags-button.png[] button located on the upper-right corner of the screen, then select *Edit Tags*.
+image:ssui-add-edit-tags.png[]
+. To add a tag, select the tag category and value from the lists, then click the blue plus (+) sign as shown in the screen capture.
+. To edit tags, simply change the value assigned to a category, or click on the cross sign in the tag to delete an existing tag.
+. Click *Save*.
+
+=== Removing a Service
+
+Follow this procedure to remove a service. 
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Remove*.
+. Alternatively, click on the service. Click the image:ssui-service-settings-button.png[] button located on the upper-right corner of the screen, then select *Remove*. A confirmation window appears.
+. Click *Yes, Remove Service*.
+
+=== Editing a Service
+
+Follow this procedure to edit the name and description of a service.
+
+. From the *Services* screen, click on more options next to the service you want to remove, then click *Edit*.
```

Comment:
[WIP] I've made this edit, but please double-check that the correct meaning is there now.